src/test/enhetstester/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vilkårsvurdering/VilkårsvurderingStegTest.kt

```diff
@@ -226,11 +265,107 @@ class VilkårsvurderingStegTest {
         every { behandlingService.hentBehandling(behandling.id) } returns behandling
         every { personopplysningGrunnlagService.hentAktivPersonopplysningGrunnlagThrows(any()) } returns personopplysningGrunnlag
         every { vilkårsvurderingService.hentAktivVilkårsvurderingForBehandling(behandling.id) } returns vilkårsvurderingForSøker
+        every { søknadGrunnlagService.hentAktiv(behandling.id) } returns søknadGrunnlagMock
+
+        val feil = assertThrows<FunksjonellFeil> {
```

Comment:
Mistenker at det er en del av oppsettet over her ☝️ Som ikke er nødvendig. Tenker spesielt på VilkårResultatene som settes opp.

Reply:
Good catch!
Jeg fjernet resultatene for barna 😊